client/client.go

```diff
@@ -110,3 +123,32 @@ func newClientTimeout(options ClientTimeout) ClientTimeout {
 
 	return ct
 }
+
+// / Generator of unique lease id
+// / Note that this Lease Id generation method may cause collisions,
+// / the client should retry after informed by the server.
+type leaseIdGenerator struct {
+	// the current lease id
+	id uint64
+}
+
+func newLeaseId() leaseIdGenerator {
+	var b [8]byte
+	_, err := rand.Read(b[:])
+	if err != nil {
+		panic(err)
+	}
+
+	var id uint64
+	atomic.StoreUint64(&id, binary.BigEndian.Uint64(b[:]))
+	return leaseIdGenerator{id: id}
```

Comment:
It seems just generate a random u64 here. Use `rand.Intn` is sufficient.

Reply:
You're right. I have followed your suggestion to modify it.